### PR TITLE
Migrate to wgpu 0.13 and fall back on fifo present mode if mailbox isn't supported in examples

### PR DESCRIPTION
**Pull request by Davidster**
## Checklist

- [x] `cargo clippy` reports no issues
- [x] `cargo doc` reports no issues
- [x] [`cargo deny`](https://github.com/EmbarkStudios/cargo-deny/) issues have been fixed or added to `deny.toml`
- [x] human-readable change descriptions added to the changelog under the "Unreleased" heading.
  - [x] If the change does not affect the user (or is a process change), preface the change with "Internal:"
  - [x] Add credit to yourself for each change: `Added new functionality @githubname`.

### CODE REVIEW

File: examples/cube.rs [hunk 1]
```diff
@@ -361,13 +361,25 @@ fn main() {
     let (device, queue) =
         block_on(adapter.request_device(&wgpu::DeviceDescriptor::default(), None)).unwrap();
 
+    let desired_present_mode = wgpu::PresentMode::Mailbox;
```
Comment (cwfitzgerald) [inline examples/cube.rs:364]: Lets just always use FIFO for the examples, no need to spin at infinite speed.

**Review by cwfitzgerald — CHANGES_REQUESTED** [review]
Good stuff, one nit but otherwise great!

**Review by cwfitzgerald — APPROVED** [review]

**Comment by Davidster** [comment]
@cwfitzgerald I see the cargo-deny job is failed but that's caused by the Cargo.lock which is gitignored. Anything I need to do to fix?

**Comment by cwfitzgerald** [comment]
Just add exceptions for those libraries that it failed for (add the _older_ versions) sometimes you get duplicates, *sigh*

**Comment by Davidster** [comment]
Done! also removed the ignored advisory since it doesn't seem to be relevant anymore

**Comment by cwfitzgerald** [comment]
Dope!

**Pull request merged**